src/h5web/dataset-visualizer/vis/matrix/IndexedGrid.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, forwardRef, Ref } from 'react';
+import { FixedSizeGrid as Grid, GridChildComponentProps } from 'react-window';
+import StickyProvider from './StickyContext';
+import styles from './MatrixVis.module.css';
+import AnchorCell from './AnchorCell';
+import IndexTrack from './IndexTrack';
+import { GridSettings } from './utils';
+
+interface Props {
+  children: (props: GridChildComponentProps) => JSX.Element;
+  itemData?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+  gridSettings: GridSettings;
+  width: number;
+  height: number;
+}
+
+// IndexedGrid based on https://codesandbox.io/s/0mk3qwpl4l
+// Wrapper around FixedSizeGrid to add indices row and column (sticky or not).
+function IndexedGrid(props: Props): JSX.Element {
+  const {
+    children: NormalCellFunc,
+    width,
+    height,
+    gridSettings,
+    itemData,
+  } = props;
+  const { cellSize, columnCount, rowCount } = gridSettings;
+
+  const [stickyState, setStickyState] = useState(true);
+  function toggleStickyIndices(): void {
+    setStickyState(!stickyState);
+  }
+
+  const InnerGrid = forwardRef(({ children: innerChildren, ...rest }, ref) => {
+    return (
+      <div ref={ref as Ref<HTMLDivElement>} {...rest}>
+        <IndexTrack
+          anchorCell
```

Comment:
The anchorCell boolean will disappear following our discussion (https://github.com/silx-kit/h5web/pull/28#discussion_r397267734)

Reply:
`AnchorCell` is now a children of `IndexTrack`. The boolean was removed.